src/components/Emoji.tsx

```diff
@@ -25,6 +18,13 @@ const Emoji = (props: IProps) => {
       options={{ protocol: IS_DEV ? "http" : "https" }}
       svg
       display="inline-block"
+      sx={{
+        "& > img": {
+          margin: "0 !important",
+          width: "1.5em !important",
+          height: "1.5em !important",
```

Comment:
Good point here!

Reply:
I've just pushed a commit to use `fontSize` on the consumer side (the `TranslationBanner` in this case) instead of hardcoding the width and height in here. See my last commit to see the changes.